MAUI/UIKit/GettingStarted.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Getting Started with MAUI UI Kit control | Syncfusion
+description: Learn here about getting started with Syncfusion Essential Studio MAUI UI Kit control, its elements and more.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Getting Started with MAUI UI Kit
+
+The UI Kit screens can be added to your application in the following two ways:
+
+1. Using **Essential UI Kit for MAUI** Visual Studio extension.
+
+2. Copying the files from our open-source [GitHub repository]().
+
+
+## Essential UI Kit for MAUI Extension
```

Comment:
Essential UI Kit for .NET MAUI Extension

Reply:
Changed In Latest Commits